src/CONFIGURATION.h

```diff
@@ -1,17 +1,28 @@
 #include "DShotRMT.h" // We need this for the dshot modes
 #include "boards_config.h" // board pinouts are in this file. you can check and modify which switch is on which pin there.
 
+//Selector settings ON BOOT, locked after booting
+uint32_t revRPMset[3][2] = { { 50000, 50000 }, { 25000, 25000 },  { 14000, 14000 } }; // adjust this to change fps, pairs are firingMode 1, 2, 3 and elements in pair are first and second stage RPM
+uint32_t idleRPMset[3][2] = { { 25000, 25000 }, { 0, 0 }, { 0, 0 } }; // adjust this to change idleRPM, pairs are firingMode 1, 2, 3 and elements in pair are first and second stage RPM
+uint32_t idleTimeSet_ms[3] = { 0, 0, 15000 }; // how long to idle the flywheels for after releasing the trigger, in milliseconds, for firingMode 1, 2, 3
+uint32_t firingDelaySet_ms[3] = {150, 75, 50}; // delay to allow flywheels to spin up before firing dart for firingMode 1, 2, 3
+
+//Live selector settings, change with switch
+uint32_t burstLengthSet[3] = { 300, 3, 1 };
+uint32_t BufferModeSet[3] = { 0, 1, 1 };
+// 0 = stop firing when trigger is released
+// 1 = complete current burst when trigger is released
+// 2 = fire as many bursts as trigger pulls
+// for full auto, set burstLength high (50+) and bufferMode = 0
+
 // Flywheel Settings
-uint32_t revRPM[4] = { 50000, 50000, 50000, 50000 }; // adjust this to change fps
-uint32_t idleRPM[4] = { 1000, 1000, 1000, 1000 };
-uint32_t idleTime_ms = 60000; // how long to idle the flywheels for after releasing the trigger, in milliseconds
 uint32_t motorKv = 3200;
 uint32_t batteryADC_mv = 14800 / 11; // battery voltage in mv divided by voltage divider ratio (11)
 dshot_mode_t dshotMode = DSHOT150; // DSHOT150 for dshot, or DSHOT_OFF to fall back to servo PWM
 
 // Dettlaff Settings
-char wifiSsid[32] = "network name";
-char wifiPass[63] = "password";
+char wifiSsid[32] = "SSID";
+char wifiPass[63] = "PASS";
```

Comment:
Why change these? It's not bad, but I don't see a need to change it

Reply:
This was just a consequence of adding and removing my own credentials, feel free to disregard

Reply:
Yeah @Jougheaux please revert